py4DSTEM/io/datastructure/py4dstem/braggvectors_fns.py

```diff
@@ -385,13 +399,22 @@ def add_indices_to_braggpeaks(
     """
     from py4DSTEM.process.latticevectors import add_indices_to_braggpeaks
 
-    bragg_peaks_indexed = add_indices_to_braggpeaks(
-        self.vectors,
-        self.braggdirections,
-        maxPeakSpacing = maxPeakSpacing,
-        qx_shift = self.Qshape[0]/2,
-        qy_shift = self.Qshape[1]/2,
-    )
+    if self.calibration.get_Q_pixel_units() == 'A^-1':
+        bragg_peaks_indexed = add_indices_to_braggpeaks(
+            self.vectors,
+            self.braggdirections_calibrated,
+            maxPeakSpacing = maxPeakSpacing * self.calibration.get_Q_pixel_size(),
```

Comment:
Would it make more sense for `maxPeakSpacing` to be in inverse angstroms in the case that the data is calibrated? Keeping it in original pixel units could be confusing